src/components/timeline/Track.css

```diff
@@ -44,32 +56,33 @@
   user-select: none;
 }
 
-.timelineThreadDetails {
-  flex: 1;
+.timelineTrackTrack {
   position: relative;
-  z-index: 1;
   display: flex;
+  flex: 1;
   flex-flow: column nowrap;
 }
 
-.timelineThreadIntervalMarkerOverview {
-  list-style: none;
-  display: block;
-  margin: 0;
-  height: 6px;
+.timelineTrackLocalTracks {
   position: relative;
-  overflow: hidden;
-  opacity: 0.75;
-}
-
-.timelineThreadIntervalMarkerOverviewContainerJank {
-  border-bottom: 1px solid var(--grey-30);
+  padding: 0;
+  list-style: none;
+  background: var(--grey-20);
 }
 
-.timelineThreadIntervalMarkerOverview.selected {
-  opacity: 1;
+.timelineTrackLocalTracks::before {
+  content: "";
+  position: absolute;
+  width: calc(100% + 15px);
```

Comment:
would be nice to have some comments explaining this magic

Reply:
Done.

Reply:
Also, to me all CSS is magic :D